setup/baseConfig.yaml

```diff
@@ -144,7 +144,7 @@ azstorage:
   update-md5: true|false <set md5 sum on upload. Impacts performance. works only when file-cache component is part of the pipeline>
   validate-md5: true|false <validate md5 on download. Impacts performance. works only when file-cache component is part of the pipeline>
   virtual-directory: true|false <support virtual directories without existence of a special marker blob>
-
+  disable-compression: true|false <disable transport layer content encoding like gzip, set this flag to true if blobs have content-encoding set in container>
```

Comment:
Should we add a test for this feature?

Reply:
we can just add a cli validation kind of a test. Data validation kind of thing can not be done as this goes to http client options and we do not have any direct control on testing it.

Reply:
Added config test and moved block blob based proxy test in code coverage to use disable-compression